app/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,47 @@
+// this has to be moved to container later, put it here because of conflicts
+.arts-container {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 100px;
+  margin-bottom: 20px;
+    a {
+      text-decoration: none;
+      color: white !important;
+    }
+}
+
+.art-card {
+  display: flex;
+  flex-direction: column;
+}
+
+.art-card-img {
+  box-sizing: border-box;
+  border-radius: 5%;
+  object-fit: cover;
+  height: 300px;
+  width: 300px;
+  margin-bottom: 5px;
+}
```

Comment:
The setup sets everything as border-box already so this can be removed.
`object-fit` is for img tags so it can be removed too.
For background-image you use background-size.
And to center the image you user background-position

Reply:
done